SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -190,7 +190,7 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 		TWeakObjectPtr<UObject> Subobject = PackageMap->GetObjectFromUnrealObjectRef(FUnrealObjectRef(Channel->GetEntityId(), SubobjectInfoPair.Key));
 		if (!Subobject.IsValid())
 		{
-			UE_LOG(LogSpatialSender, Error, TEXT("Tried to generate initial replication state for an invalid sub-object. Object may have been deleted or is PendingKill."));
+			UE_LOG(LogSpatialSender, Warning, TEXT("Tried to generate initial replication state for an invalid sub-object (class %s, sub-object %s, actor %s). Object may have been deleted or is PendingKill."), *SubobjectInfo->Class->GetName(), *SubobjectInfo->SubobjectName.ToString(), *Actor->GetName());
```

Comment:
`SubobjectInfo` can be null, no?

Reply:
No.

Reply:
To be more clear: `SubobjectInfo` map that is being iterated above only contains valid `FClassInfo*`. I am adding a change to https://github.com/spatialos/UnrealGDK/pull/596 that will make those `FClassInfo&` so it will be obvious they're always valid.